### DISCUSSION TITLE
Dr. Agus Sudjianto's Retirement from Head of MRM at Wells Fargo

### DISCUSSION THREAD

**Discussion by ajzhanghk** [SelfExplainML/PiML-Toolbox#50] [General]
Screenshots of LinkedIn post w/ comments (posted on Jan 29, 2024; screenshots captured on Jan 31, 2024). 

![Image_20240131165512](https://github.com/SelfExplainML/PiML-Toolbox/assets/25238259/81b4ddbd-da81-46e3-a7a9-22e0a00c7789)

![Image_20240131165212](https://github.com/SelfExplainML/PiML-Toolbox/assets/25238259/47366c31-9f81-44ae-ac67-f45934343e22)

![Image_20240131165221](https://github.com/SelfExplainML/PiML-Toolbox/assets/25238259/1a492afc-0ebc-4475-9c57-7851501fe806)

![Image_20240131151430](https://github.com/SelfExplainML/PiML-Toolbox/assets/25238259/0dd09957-2b38-46a3-b72e-5dab1c140261)

![Image_20240131151748](https://github.com/SelfExplainML/PiML-Toolbox/assets/25238259/17035096-9a40-4483-9d48-c84d03835c59)

**PS:** The original post has to be deleted due to an inappropriate sharing of internal-use-only announcement. Apologies to all the friends who warmly commented and reacted with likes and loves: 
[Agus Sudjianto](https://www.linkedin.com/in/agus-sudjianto-76519619/), 
[Arpit Narain, CFA, FRM, CQF](https://www.linkedin.com/in/arpit-narain-cfa-frm-cqf/), 
[Collin Starkweather](https://www.linkedin.com/in/collinstarkweather/), 
[Xiaoming Li](https://www.linkedin.com/in/xiaoming-li-08363b27/), 
[Emily Riederer](https://www.linkedin.com/in/emilyriederer/), 
[Nguyen Hieu Trung](https://www.linkedin.com/in/nguyen-hieu-trung-a5a836213/), 
[Joe H.](https://www.linkedin.com/in/computer-vision-engineer/), 
[James Oche](https://www.linkedin.com/in/james-oche-33112a/), 
[Sunil Padture](https://www.linkedin.com/in/sunil-padture/), 
[Simone Flory](https://www.linkedin.com/in/simone-flory/),
[Satyaki G.](https://www.linkedin.com/in/satyakigd/),
[Nikolai Kukharkin](https://www.linkedin.com/in/nikolai-kukharkin/),
[Girish Govind](https://www.linkedin.com/in/girish-govind-250b502/), 
[Rafic Fahs](https://www.linkedin.com/in/rafic-fahs-90a47613/),
[Carlos A. Herrera, CFA, FRM, FDP](https://www.linkedin.com/in/carlosherreracfafrm/),
[Ruiying (Diana) C.](https://www.linkedin.com/in/ruiying-diana-c-71778016/),
[Shameek Kundu](https://www.linkedin.com/in/shameekkundu/),
[Vikas Tyagi](https://www.linkedin.com/in/vikas-tyagi-6989974/),
[Viswaram (Ram) Janakiraman](https://www.linkedin.com/in/viswaram-ram-janakiraman-6609194/),
[Taposh Dutta Roy](https://www.linkedin.com/in/taposh/),
[Daniel Saunders](https://www.linkedin.com/in/daniel-saunders-10530261/),
[Srikanth Ranganathan](https://www.linkedin.com/in/srikanth-ranganathan-2272b82/),
[Raphaella Lancia](https://www.linkedin.com/in/raphaellalancia/),
[Sayan Datta](https://www.linkedin.com/in/sayandatta/),
[Balaji SR](https://www.linkedin.com/in/balaji-sr/),
[Jeremy B. Smith](https://www.linkedin.com/in/jeremy-b-smith-9500156/),
[Imir Arifi PhD](https://www.linkedin.com/in/imir-arifi-phd-1284139/),
[Akhil Vakayil](https://www.linkedin.com/in/avkl/),
[Matthew A. Campbell](https://www.linkedin.com/in/matthewalancampbell/),
[Jacob Kosoff](https://www.linkedin.com/in/jacob-kosoff/),
[Jameson Augustin](https://www.linkedin.com/in/jameson-augustin-462a5317b/),
[zhanyang zhang](https://www.linkedin.com/in/zhanyang-zhang-1086141a5/),
[Chris Smigielski](https://www.linkedin.com/in/chris-smigielski-bb718b57/),
[Jackie Holmes](https://www.linkedin.com/in/jackie-holmes-87103310/),
[Mandar Kedari](https://www.linkedin.com/in/mandar-kedari-154b593a/),
[Patricio Morat](https://www.linkedin.com/in/patriciomorat/),
[Huy Tran](https://www.linkedin.com/in/huy-vo-tran/),
[Ling L.](https://www.linkedin.com/in/lingliu-sf/),
[Austin Saragih](https://www.linkedin.com/in/austinsaragih/),
[Arsa Oemar](https://www.linkedin.com/in/arsa-oemar/),
[Michel Hantar](https://www.linkedin.com/in/michel-hantar-a789673/)
[Robert Vander Hooning](https://www.linkedin.com/in/robertvan/)

**Comment by ajzhanghk** [comment]
This original post is reshared in LinkedIn (after removal of the internal announcement info) with URL https://www.linkedin.com/feed/update/urn:li:activity:7158579650695208960/

**Comment by BradKML** [comment]
STOP. USING. GitHub. FOR. PROMOTION.
PLEASE.
